packages/backend/src/modules/attachment/attachments.controller.ts

```diff
@@ -0,0 +1,39 @@
+import express from 'express'
+import { AttachmentDTO } from 'shared'
+import { Attachment } from './Attachment'
+import { AttachmentService, attachmentService } from './attachment.service'
+import { AttachmentMapper, attachmentMapper } from './attachment.mapper'
+
+class AttachmentsController {
+  private readonly _attachmentMapper: AttachmentMapper
+  private readonly _attachmentService: AttachmentService
+
+  constructor(attachmentService: AttachmentService, attachmentMapper: AttachmentMapper) {
+    this._attachmentMapper = attachmentMapper
+    this._attachmentService = attachmentService
+  }
+
+  async create(req: any, res: express.Response) {
+    const createdAttachment: Attachment = await this._attachmentService.create(req.file)
```

Comment:
we need taskId to know to which task this attachement belongs use:
`interface CreateAttachmentDTO {
  taskId: string
  file: File
}`